app/gcl/associate-cloud-engineer/section3/page.module.css

```
@@ -0,0 +1,1037 @@
+    font-size: 15px;
+    font-weight: 700;
+    color: #00c8ff;
+    margin-top: 2px;
+}
+
+.ace-section3-page :global(.app-layout) {
+    display: block;
+    min-height: 100vh;
+}
+
+.ace-section3-page :global(.main-content) {
```

Biome (2.4.16)

[error] Unexpected unknown pseudo-class global 

(lint/correctness/noUnknownPseudoClass)